.github/workflows/securitypipeline.yaml

```diff
@@ -58,6 +66,15 @@ jobs:
         with:
           command: test
           args: --org=plan-x --severity-threshold=high --fail-on=upgradable
+      - name: comment on PR
+        # comment on PR if snyk found vulnerabilities
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.number }}
+          body: |
+            Snyk found vulnerabilities in this PR
```

Comment:
do we really need to post on the PR? since the message doesn't contain details about the failure, maybe it's enough that the check on the commit fails?

Reply:
That is fair I added that in the case that if the step found vulns, it would then comment,  it is possible, for example if you haven't moved off requests.txt and didn't specify no poetry the step will still fail. But this is totally a fair comment so I have removed it for now and we can come back if it becomes necessary.